### PR TITLE
Add option and subroom

### PR DESCRIPTION
**Pull request by aUsefulMethod**
Wanted to continue the story and see where our creativity will take us. changed 3-1-1A and created 3-2.

### CODE REVIEW

File: english/normal-dungeon/3/2.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+As you approach your home, the 🚪door is ajar, ever so slightly, 🕯candlight emanating from the cracks.
```
Comment (ming-tsai) [inline english/normal-dungeon/3/2.md:1]: Hi @aUsefulMethod, nice story but the options are not pointing to other rooms

Comment (aUsefulMethod) [reply english/normal-dungeon/3/2.md:1]: I was going to leave those open if anyone wanted to hop in and create the next part (sub-rooms). I can fix that and make them point to existing rooms if you want?

Comment (ming-tsai) [reply english/normal-dungeon/3/2.md:1]: Yes

Comment (aUsefulMethod) [reply english/normal-dungeon/3/2.md:1]: Ok, I added fix :)

Comment (ming-tsai) [reply english/normal-dungeon/3/2.md:1]: Nice work 👍

**Pull request merged**